src/daemon-base/Dockerfile

```diff
@@ -6,6 +6,7 @@ MAINTAINER __DOCKERFILE_MAINTAINER__
 __DOCKERFILE_TRACEABILITY_LABELS__
 
 ENV CEPH_VERSION __ENV_[CEPH_VERSION]__
```

Comment:
As a suggestion (but I'm not hard-set), it could be useful to comment these so the user might have a better idea of what to expect. Like:

```
# Named Ceph version. e.g., luminous, mimic, ...
ENV CEPH_VERSION ...
# Specific Ceph version with a package version specifier symbol prepended (may be unset w/ empty string)
# e.g. -12.3.0<I'm not sure what the rest of the centos text is>, =12.3.0

Reply:
That duplicates the information in many places ... we have a good documentation. So let's use it ;)